lib/dl_api_lib_testing/dl_api_lib_testing/connector/dataset_suite.py

```diff
@@ -33,3 +38,23 @@ def test_remove_connection(
         sync_us_manager.delete(sync_us_manager.get_by_id(saved_connection_id))
         dataset_resp = control_api.load_dataset(saved_dataset)
         assert dataset_resp.status_code == 200, dataset_resp.json
+
+    def test_dataset__delete_result_field__available_after_refresh(
```

Comment:
imho, a bit too long of a name, one can understand the details of the test by reading it's body. i'd suggest `test_refresh_sources`

Reply:
I disagree with `test_refresh_sources`, because in this case we don't test like can we refresh or not. In this particular case we test if field is available after refresh.
imho: I d better read test name and not read its body so regarding tests long names are appropriate

Reply:
imho, it only makes logs slightly less readable, tho it's not worth arguing for)